anchore_engine/services/policy_engine/engine/feeds/feeds.py

```diff
@@ -921,21 +956,19 @@ def _process_group_file_records(
                 total_records_updated += 1
                 logger.info(
                     self._log_context.format_msg(
-                        "DB Update Progress: {}/{}".format(
-                            total_records_updated,
-                            group_download_result.total_records,
-                        ),
-                    )
+                        "DB Update Progress: %d/%d"
+                        % (total_records_updated, group_download_result.total_records)
```

Comment:
I'm really confused. I changed the string format here to the % interpolation style because of feedback from @zburstein that using .format was unsafe. There has been a lot of back-and-forth between folks on the team as to how to format strings, and I don't understand what is preferred or why. Would really appreciate some clarity.

Reply:
Yeah we had a discussion with @nightfurys about how we are doing logging and if Im not mistaken she said that using the percent format was the best way to interpolate with our logger. Is there an alternative way that you suggest?